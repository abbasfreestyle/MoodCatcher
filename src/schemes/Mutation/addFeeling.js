const addFeeling = `
  mutation(
    $name: String!,
    $feelingMoodId: ID!
  ) {
    createFeeling(input:{
      name: $name,
      feelingMoodId: $feelingMoodId
    }) {
      id
      name
    }
  }
`;

export default addFeeling;
