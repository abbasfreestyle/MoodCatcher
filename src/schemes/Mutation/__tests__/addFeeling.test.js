import addFeeling from '../addFeeling';

describe('Schemes > Mutation > Add Feeling', () => {
  test('should match the snapshot', () => {
    expect(addFeeling).toMatchSnapshot();
  });
});
