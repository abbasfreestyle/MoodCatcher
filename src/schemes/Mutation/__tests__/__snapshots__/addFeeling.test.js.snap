// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Schemes > Mutation > Add Feeling should match the snapshot 1`] = `
"
  mutation(
    $name: String!,
    $feelingMoodId: ID!
  ) {
    createFeeling(input:{
      name: $name,
      feelingMoodId: $feelingMoodId
    }) {
      id
      name
    }
  }
"
`;
